<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kamer_jesper</class>
 <widget class="QWidget" name="kamer_jesper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="kamer_naam">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>98</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Jesper's kamer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deur">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>de deur is open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kast">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="text">
    <string>kast</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geheime_deur">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>geheime deur</string>
   </property>
  </widget>
  <widget class="QToolButton" name="deur_knop">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>deur knop</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>20</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>18</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QPushButton" name="muur_links_onder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>41</width>
     <height>171</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>muur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="muur_links_boven">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>muur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="muur_rechts_boven">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>muur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="muur_rechts_onder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>41</width>
     <height>171</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>muur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geheime_deur_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>21</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>ge
he
im
e
de
ur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kist">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>kist</string>
   </property>
  </widget>
  <widget class="QLabel" name="deur_knop_hint">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;HÃ¨, hij schiet meteen weer
dicht. Had ik maar een tak ofzo.&quot;</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="antwoord_op_vraag">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="vraag">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>wil je je tak tegen de
 deurknop aan drukken</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="vraag_of_tak_terug">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
